docs/static/spec/openapi/logstash-api.yaml

```
@@ -346,13 +654,91 @@ paths:
+                                  properties:
+                                    status:
+                                      $ref: '#/components/schemas/Status'
+                                    symptom:
+                                      type: string
+                                      description: "A message providing information about the current health status."
+                                      example: "The pipeline is healthy"
+                                    details:
+                                      type: object
+                                      description: "An object that contains additional information about the indicator that has lead to the current health status result."
+                                      properties:
+                                        status:
+                                          type: object
+                                          properties:
+                                            state:
+                                              type: string
+                                              enum:
+                                                - LOADING
+                                                - RUNNING
+                                                - FINISHED
+                                                - TERMINATED
+                                        flow:
+                                          type: object
+                                          properties:
+                                            worker_utilization:
+                                              $ref: '#/components/schemas/FlowWindows'
+                                    impacts:
+                                      type: array
+                                      description: "If a non-healthy status is returned, indicators may include a list of impacts that this health status will have on Logstash."
+                                      items:
+                                        type: object
+                                        properties:
+                                          severity:
+                                            type: integer
+                                            minimum: 1
+                                            maximum: 10
+                                            description: "How important this impact is to the functionality of Logstash. A value of 1 is the highest severity, with larger values indicating lower severity."
+                                          description:
+                                            type: string
+                                            description: "A description of the impact on Logstash"
+                                          impact_areas:
+                                            type: array
+                                            description: "The areas Logstash functionality that this impact affects. Possible values are:"
```

I'm not sure I understand this context entirely but I think it needs clarification. Something like this?
```suggestion
                                            description: "The areas of Logstash affected by the health status. Possible values are:"
```